json-rpc/src/transactions.rs

```diff
@@ -0,0 +1,148 @@
+use crate::views::{
```

Comment:
nit (code organization): I'm not sure what's a reason to have this separate file
IMO `get_account_transaction` and `get_transactions` can be defined in methods.rs
And `TryInto` fits well at the bottom of views.rs

Reply:
fixed.